src/Sulu/Bundle/AdminBundle/Resources/js/components/Schedules/WeeklySchedule.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import React from 'react';
+import {transformDateToTimeString, transformTimeStringToDate} from '../../utils/Date';
+import {translate} from '../../utils/Translator';
+import DatePicker from '../DatePicker';
+import MultiSelect from '../MultiSelect';
+import Form from '../Form';
+import type {Weekday, WeeklyScheduleEntry} from './types';
+
+type Props = {|
+    index: number,
+    onChange: (index: number, value: WeeklyScheduleEntry) => void,
+    value: WeeklyScheduleEntry,
+|};
+
+export default class WeeklySchedule extends React.Component<Props> {
+    handleDaysChange = (days: Array<Weekday>) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, days});
+    };
+
+    handleStartChange = (start: ?Date) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, start: start ? transformDateToTimeString(start) : undefined});
+    };
+
+    handleEndChange = (end: ?Date) => {
+        const {index, onChange, value} = this.props;
+        onChange(index, {...value, end: end ? transformDateToTimeString(end) : undefined});
+    };
+
+    render() {
+        const {value} = this.props;
+
+        return (
+            <Form>
+                <Form.Field colSpan={6} label={translate('sulu_admin.weekdays')} spaceAfter={6}>
+                    <MultiSelect onChange={this.handleDaysChange} values={value.days || []}>
```

Comment:
we should set the `noneSelectedText` to tell the user that the times are effective for all days per default

Reply:
What would you suggest as a text? Writing "All selected" or something like that although nothing is selected seems also confusing to me.

Reply:
Would ust use `Everyday` as `allSelectedText` and also `noneSelectedText` 🙂

Reply:
Still think that this is confusing... Since you have already suggested using default values below, I would say we do the same here, and make all days selected by default.

Reply:
is okay for me too, but then we should only match the selected days on the server and do not implement a special handling for an empty selection.

Reply:
Made it a mandatory field and set all days by default.